src/pcrlock/pcrlock.c

```diff
@@ -1927,6 +1927,9 @@ static int event_log_map_components(EventLog *el) {
                         continue;
                 }
 
+                if (!c->n_variants)
+                        log_notice("%s has no variant\n", c->id);
```

Comment:
Our log message functions do not need a \n suffix, please remove.

Also, say what we are talking about here, i.e.:

```c
log_notice("Note: Component '%s' has no defined variants.");
```